src/Toolkit/Obj.php

```diff
@@ -64,12 +65,24 @@ public function __get(string $property)
     /**
      * Property Getter
      *
-     * @param string $property
+     * @param string|array $property
      * @param mixed $fallback
      * @return mixed
      */
-    public function get(string $property, $fallback = null)
+    public function get(string|array $property, $fallback = null)
```

Comment:
Union type hints were introduced in PHP 8, we are not quite there yet:

```suggestion
    public function get($property, $fallback = null)
```